ipywidgets/static/widgets/less/widgets.less

```diff
@@ -178,37 +180,34 @@
     }
 }
 
+.widget-colorpicker {
+    /* Button */
+    width : @widget-width-short;
+}
+
 .widget-button {
     /* Button */
-    width         : @widget-width-short;
+    width : @widget-width-short;
 }
 
 .widget-text {
     /* Textbox */
-    width  : @widget-width;
-    margin : 0px;
+    width : @widget-width;
 }
 
 .widget-textarea {
     /* TextAreaView */
-    width  : @widget-width;
-    margin : 0px;
+    width : @widget-width;
 }
 
 .widget-listbox {
     /* Listbox */
-    width         : @widget-width;
-    margin-bottom : 0px;
-    /* Bootstrap does some weird stuff with the margins of select boxes.  The 
-    specifity of the media selector needs to be overwritten using important */
-    margin-left   : 0px !important; 
-    margin-right  : 0px !important;
+    width : @widget-width;
```

Comment:
Your change here was removed in the rebase. I can ad it back I guess but, it should probably be `2px`.